src/Sarif/NotYetAutoGenerated/AddressComparer.cs

```diff
@@ -1,113 +0,0 @@
-// Copyright (c) Microsoft.  All Rights Reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using System.CodeDom.Compiler;
-using System.Collections.Generic;
-
-namespace Microsoft.CodeAnalysis.Sarif
-{
-    /// <summary>
-    /// Defines methods to support the comparison of objects of type Address for sorting.
-    /// </summary>
-    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "1.1.5.0")]
-    internal sealed class AddressComparer : IComparer<Address>
-    {
-        internal static readonly AddressComparer Instance = new AddressComparer();
-
-        public int Compare(Address left, Address right)
-        {
-            int compareResult = 0;
-
-            // TryReferenceCompares is an autogenerated extension method
-            // that will properly handle the case when 'left' is null.
-            if (left.TryReferenceCompares(right, out compareResult))
-            {
-                return compareResult;
-            }
-
-            compareResult = left.AbsoluteAddress.CompareTo(right.AbsoluteAddress);
-            if (compareResult != 0)
-            {
-                return compareResult;
-            }
-
-            // TryReferenceCompares is an autogenerated extension method
-            // that will properly handle the case when 'left' is null.
-            if (left.RelativeAddress.TryReferenceCompares(right.RelativeAddress, out compareResult))
-            {
-                return compareResult;
-            }
-
-            compareResult = left.RelativeAddress.Value.CompareTo(right.RelativeAddress.Value);
-            if (compareResult != 0)
-            {
-                return compareResult;
-            }
```

Comment:
1. I found a bug here, if both left and right are null, TryReferenceCompares will directly return true and all the other properties are not compared down below.

2. The fix by adding CompareTo<T>not only fix this bug but also able to remove this file entirely.